ansible/roles_ocp_workloads/ocp4_workload_quay_workshop/tasks/workload.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Determine Cluster Base Domain for Quay Route
+  k8s_info:
+    api_version: config.openshift.io/v1
+    kind: Ingress
+    name: cluster
+  register: r_ingress_config
+
+# Example return: CLUSTER_SUBDOMAIN = apps.cluster-84d2.84d2.example.opentlc.com
+- name: Store the subdomain
+  set_fact:
+    CLUSTER_SUBDOMAIN: "{{ r_ingress_config.resources[0].spec.domain }}"
+
+- name: Create Quay Operator NS, Homeroom NS, rolebinding, and secret
+  k8s:
+    state: present
+    definition:  "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+    - ./templates/quay-ns.yml
+    - ./templates/homeroom-ns.yml
+    - ./templates/rolebinding.yml
+    - ./templates/operatorgroup.yml
+    - ./templates/subscription.yml
+    - ./templates/secret.yml
```

Comment:
No need to include `./templates/`, the templates directory is already implied for the lookup path when using `lookup('template', ...)`